ravenframework/Samplers/DynamicEventTree.py

```diff
@@ -152,6 +152,7 @@ def __init__(self):
     self.standardDETvariables              = []
     # Dictionary of variables that represent the epistemic space (hybrid det). Format => {'epistemicVarName':{'HybridTree name':value}}
     self.epistemicVariables                = {}
+    self.fullyCorrelatedEpistemicToVar     = {}
```

Comment:
these .self variables should be described for clarity

Reply:
added description